src/software/utils/main_imageProcessing.cpp

```diff
@@ -801,7 +797,6 @@ int aliceVision_main(int argc, char * argv[])
                 options.rawColorInterpretation = rawColorInterpretation;
             }
             options.colorProfileFileName = view.getColorProfileFileName();
-            options.applyToneCurve = applyToneCurve;
```

Comment:
Should it not also be removed from ImageReadOptions?

Reply:
Done